go/kbfs/libkbfs/dirty_bcache_disk.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2018 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"context"
+	"sync"
+
+	"github.com/keybase/client/go/kbfs/tlf"
+	"github.com/pkg/errors"
+)
+
+type dirtyBlockCacheDiskConfig interface {
```

Comment:
Why here and not in `interfaces.go`?

Reply:
This is a pattern we use when specifying a subset of the large `Config` interface, which includes only the exact items that will be used by the service.  We do this in several other places, and it's always defined alongside the struct that uses it.  I guess since it's only used in one place, it's easier to read the code that way, as opposed to putting it in a different file.